core/src/main/resources/jenkins/model/Jenkins/projectRelationship.jelly

```diff
@@ -28,9 +28,15 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <l:layout title="${%Project Relationship}">
+    <l:app-bar title="${title}">
+      <a class="jenkins-button" tooltip="${%Help}" href="projectRelationship-help">
+        <l:icon src="symbol-help-circle" class="icon-sm" />
+      </a>
+    </l:app-bar>
```

Comment:
Unnecessary unrelated structural change? Why not fix up the icon inside `h1` and done?